js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-results.js

```diff
@@ -132,6 +135,14 @@ define([
             return momentAPI.formatDateTimeUTC(date);
         }
 
+        updateThreshold() {
+            this.loadData();
+        }
+
+        resultCountText() {
+            return 'Viewing First ' + this.resultsCountFiltered() + ' of ' + this.resultsCount() + ' records';
```

Comment:
Why not templated string?
`` `Viewing First ${this.resultsCountFiltered()} of ${this.resultsCount()} records` ``

Reply:
done